cmd/connector/conn_validate.go

```diff
@@ -43,9 +48,20 @@ func newConnValidateCmd(apiClient client.Client) *cobra.Command {
 			check := cmd.Flags().Lookup("check").Value.String()
 
 			isReadOnly, _ := strconv.ParseBool(cmd.Flags().Lookup("read-only").Value.String())
+			readLimit := cmd.Flags().Lookup("read-limit").Value.String()
```

Comment:
I think we might be able to use the `GetInt()` helper on `Flags()` instead of converting it to an integer ourselves.

Reply:
Checking

Reply:
Done